Sources/OpenTelemetryApi/Trace/SemanticAttributes.swift

```diff
@@ -12,141 +12,1494 @@
+    */
+    public enum FaasTriggerValues: String {
+        /**
+        A response to some data source operation such as a database or filesystem read/write.
+        */
+        case datasource = "datasource"
+        /**
+        To provide an answer to an inbound HTTP request.
+        */
+        case http = "http"
+        /**
+        A function is set to be executed when messages are sent to a messaging system.
+        */
+        case pubsub = "pubsub"
+        /**
+        A function is scheduled to be executed regularly.
+        */
+        case timer = "timer"
+        /**
+        If none of the others apply.
+        */
+        case other = "other"
+    }
+    
+    /**
+    Describes the type of the operation that was performed on the data.
+    */
+    public struct FaasDocumentOperationValues: CustomStringConvertible {
+        /**
+        When a new object is created.
+        */
+        static let insert = FaasDocumentOperationValues("insert")
+        /**
+        When an object is modified.
+        */
+        static let edit = FaasDocumentOperationValues("edit")
+        /**
+        When an object is deleted.
+        */
+        static let delete = FaasDocumentOperationValues("delete")
+
+        internal let value: String
+
+        public init(_ customValue: String) {
+            self.value = customValue
+        }
+
+        public var description: String {
+            return value
+        }
+    }
+    
+    /**
+    Kind of HTTP protocol used.
+    */
+    public struct HttpFlavorValues: CustomStringConvertible {
+        /**
+        HTTP 1.0.
+        */
+        static let HTTP10 = HttpFlavorValues("1.0")
+        /**
+        HTTP 1.1.
+        */
+        static let HTTP11 = HttpFlavorValues("1.1")
+        /**
+        HTTP 2.
+        */
+        static let HTTP20 = HttpFlavorValues("2.0")
+        /**
+        SPDY protocol.
+        */
+        static let SPDY = HttpFlavorValues("SPDY")
+        /**
+        QUIC protocol.
+        */
+        static let QUIC = HttpFlavorValues("QUIC")
+
+        internal let value: String
+
+        public init(_ customValue: String) {
+            self.value = customValue
+        }
+
+        public var description: String {
+            return value
+        }
+    }
+    
+    /**
+    The kind of message destination.
+    */
+    public enum MessagingDestinationKindValues: String {
+        /**
+        A message sent to a queue.
+        */
+        case queue = "queue"
+        /**
+        A message sent to a topic.
+        */
+        case topic = "topic"
+    }
+    
+    /**
+    The cloud provider of the invoked function.
+    */
+    public struct FaasInvokedProviderValues: CustomStringConvertible {
+        /**
+        Amazon Web Services.
+        */
+        static let AWS = FaasInvokedProviderValues("aws")
+        /**
+        Amazon Web Services.
+        */
+        static let Azure = FaasInvokedProviderValues("azure")
```

Comment:
Mistake in the specification - PR here: https://github.com/open-telemetry/opentelemetry-specification/pull/1595